devices/AD5328/AD5328.nfproj

```diff
@@ -80,4 +81,11 @@
     </PropertyGroup>
     <Error Condition="!Exists('packages\Nerdbank.GitVersioning.3.5.103\build\Nerdbank.GitVersioning.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Nerdbank.GitVersioning.3.5.103\build\Nerdbank.GitVersioning.targets'))" />
   </Target>
-</Project>
\ No newline at end of file
+  <Import Project=".\packages\StyleCop.MSBuild.6.2.0\build\StyleCop.MSBuild.targets" Condition="Exists('.\packages\StyleCop.MSBuild.6.2.0\build\StyleCop.MSBuild.targets')" />
```

Comment:
These paths look better without starting with `.\` don't they?

Reply:
For me, it's the same, but changed it anyway.

Reply:
Sure! Just that all the others are using the "no dot" notation and thought of making this consistent. Picky of me... 😅